google-cloud-video_intelligence/Rakefile

```diff
@@ -154,6 +179,16 @@ task :ci do
   header "google-cloud-video_intelligence test", "*"
   sh "bundle exec rake test"
 end
+
+namespace :ci do
```

Comment:
Why is the task `ci:smoke_test` being added? Is this supposed to be used somewhere?

Reply:
This is spurious. Filed https://github.com/googleapis/toolkit/issues/1750.